Automattic/jetpack
# Protect: Update `Navigation` styles and logic to be controlled by consumer

This move `Navigation` to be controlled from Consumer, since it's base usage for it at `VulnerabilitiesList`.

Related with #24064 

#### Changes proposed in this Pull Request:

* Move `Navigation` selected to be defined by the consumer.
* Fix margin issues.

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start `Storybook`
* Navigate to `Protect/Components/Navigation`.
* It should work normally (click and keyboard).


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202117981194312
